blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxWriter.java

```diff
@@ -134,7 +148,12 @@ public void insertFull(Collection<? extends IMetric> metrics) throws ConnectionE
                     AstyanaxWriter.setLocatorCurrent(locator);
                 }
 
+                if (!isString && !isBoolean && mutationBatch != null) {
```

Comment:
If there isn't a `mutationbatch` should we create one?  Looks like there isn't a check around `insertMetric(metric, mutationBatch)` below.